parsers/to-style-dictionary/README.md

```diff
@@ -0,0 +1,457 @@
+| `formatTokens.fontFormat`          | optional | `woff2` `woff` `otf` `ttf` `eot`                                  | `ttf`       | The formats of font files you use                                                                                                                                                                          |
+| `splitBy`                          | optional | `string`                                                          |             | The character used to define the nesting of the values in the object (e.g. The name of the color in [this example](https://github.com/Specifyapp/parsers/tree/master/parsers/to-style-dictionary#input-2)) |
+| `assetsBaseDirectory.fonts`        | optional | `string`                                                          | `none`      | the base directory of your font assets use                                                                                                                                                                 |
+| `assetsBaseDirectory.images`       | optional | `string`                                                          | `none`      | the base directory of your image assets use                                                                                                                                                                |
+| `assetsBaseDirectory.icons`        | optional | `string`                                                          | `none`      | the base directory of your icon assets use                                                                                                                                                                 |
+| `formatConfig.endOfLine`           | optional | `auto` `lf` `crlf` `cr`                                           | `auto`      | [Prettier documentation](https://prettier.io/docs/en/options.html#end-of-line)                                                                                                                             |
+| `formatConfig.tabWidth`            | optional | `number`                                                          | `2`         | [Prettier documentation](https://prettier.io/docs/en/options.html#tab-width)                                                                                                                               |
+| `formatConfig.useTabs`             | optional | `boolean`                                                         | `true`      | [Prettier documentation](https://prettier.io/docs/en/options.html#tabs)                                                                                                                                    |
+
+## Types
+
+ℹ️ **Please be aware that, depending on the order you use parsers, their input and output types have to match.**
+
+### Input
+
+Array of object with at least name, value and type:
```

Comment:
Array of objects containing at least the following properties: `name`, `value` and `type`: